app/src/main/java/com/github/h3lp3rs/h3lp/HelpPageActivity.kt

```diff
@@ -81,7 +81,10 @@ class HelpPageActivity : AppCompatActivity(), CoroutineScope by MainScope() {
         apiHelper = GoogleAPIHelper(resources.getString(R.string.google_maps_key))
 
         // Initialize the current user's location
-        setupLocation()
+        locationHelper.requireAndHandleCoordinates(this) {
+            currentLat = it.latitude
+            currentLong = it.longitude
```

Comment:
If you make the suggested change to have 2 public variables in LocationHelper, there would be no need for currentLat and currentLong, you could just change their uses to locationHelper.latitude and locationHelper.longitude